recipes/requestsexceptions/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{%set name = "requestsexceptions" %}
+{%set version = "1.1.3" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d678b872f51f76d875e00e6667f4ddbf013b3a99490ae5fe07cf3e4f846e283e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
```

Comment:
@pmlandwehr this is ultimately your decision as you will be maintaining the feedstock, but all that `jinja` might get in the way instead of helping. It will take some time for others to read the recipe and get what is going on. For example: if someone needs to copy-n-paste the URL to download the source and create a patch or check/change the checksum?